extension.neon

```diff
@@ -39,3 +39,9 @@ parameters:
         - wp_send_json_error
         - wp_nonce_ays
         - dead_db
+    earlyTerminatingMethodCalls:
+        \WP_CLI:
+            - WP_CLI::error
+            - WP_CLI::halt
+            - WP_CLI::launch
+            - WP_CLI::runcommand
```

Comment:
I think these commands terminate execution only with a complex condition.

https://github.com/wp-cli/wp-cli/blob/d7df11b43e9ece6990ac42371aadac2cbd9d6554/php/class-wp-cli.php#L1270-L1271

`earlyTerminatingMethodCalls` is for methods unconditionally terminating execution.

Even if you write a PHPStan service for `WP_CLI::runcommand` you cannot figure out what the command's exit status will be.
:) So never enable `exit_error` to keep your code testable.

Reply:
Added this warning to the README